cime_config/tests.py

```diff
@@ -305,6 +305,7 @@
             "SMS_D_Ld1.ne30pg2_r05_EC30to60E2r2.WCYCL1850",
             "SMS_Ln5.ne30pg2_ne30pg2.F2010-SCREAM-LR-DYAMOND2",
             "ERS_Ld3.ne30pg2_r05_EC30to60E2r2.WCYCL1850.allactive-nlmaps",
+            "SMS_D_Ld1.ne30pg2_IcoswISC30E3r5.CRYO1850",
```

Comment:
@jonbob, does this look like what you had in mind at our meeting today?

Reply:
I think so -- unless you want to test CRYO1850-DISMF? I'm somehow especially aware these days of the difference between the two! Plus maybe the trigrid version, since it's what we are using more by default

Reply:
@jonbob, I think we should probably test both DISMF and PISMF (the default), so maybe:
```suggestion
            "SMS_D_Ld1.ne30pg2_r05_IcoswISC30E3r5.CRYO1850",
            "SMS_D_Ld1.ne30pg2_r05_IcoswISC30E3r5.CRYO1850-DISMF",
```